lookout/style/format/benchmarks/top_repos_quality.py

```diff
@@ -231,6 +233,7 @@ def calc_avg(arr: Sequence[Sequence], col: int) -> float:
     ("recall", float),
     ("full_recall", float),
     ("f1", float),
+    ("full_f1", float),
```

Comment:
I think it is worth to make a comment about `full_` prefix because It requires some time for me to remember what it means. 
I suggest to add something like:
```python
# Metrics for the quality report. Metrics are calculated on the samples subset 
# where predictions were made.  `full_` prefix means that metric was calculated 
# on all available samples. `ppcr` means predicted positive condition rate and
# shows the ratio of samples where the model was able to predict. 
```

before `NamedTuple` defenition.